Doc/c-api/init.rst

```diff
@@ -359,10 +359,11 @@ Process-wide parameters
    (converted to wide characters).
    This is used by :c:func:`Py_GetPath` and some other functions below to find
    the Python run-time libraries relative to the interpreter executable.  The
-   default value is ``'python'``.  The argument should point to a
-   zero-terminated wide character string in static storage whose contents will not
-   change for the duration of the program's execution.  No code in the Python
-   interpreter will change the contents of this storage.
+   default value is ``'python'`` on Windows and ``'python3'`` elsewhere.  The
+   argument should point to a zero-terminated wide character string in static
+   storage whose contents will not change for the duration of the program's
+   execution.  No code in the Python interpreter will change the contents of
+   this storage.
```

Comment:
I suggest to link to https://docs.python.org/dev/c-api/init_config.html#c.PyConfig.program_name which documents the default value.

Reply:
Good point. I extended the text with a bit more detail. This includes removing the 'static lifetime' remark. Please let me know if that should be reinstated.